<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolboxGOMarkov</class>
 <widget class="QWidget" name="ToolboxGOMarkov">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>182</width>
        <height>426</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonPointer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Pointer</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_pointer.png</normaloff>:/icon/resource/image/tool_pointer.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSelect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_select.png</normaloff>:/icon/resource/image/tool_select.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Text</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_text.png</normaloff>:/icon/resource/image/tool_text.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonEquivalent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Equivalent</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_equivalent.png</normaloff>:/icon/resource/image/tool_equivalent.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Common Cause</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_common.png</normaloff>:/icon/resource/image/tool_common.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Two State</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op1.png</normaloff>:/icon/resource/image/tool_op1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>OR</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op2.png</normaloff>:/icon/resource/image/tool_op2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Trigger</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op3.png</normaloff>:/icon/resource/image/tool_op3.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Signal</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op5.png</normaloff>:/icon/resource/image/tool_op5.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op6.png</normaloff>:/icon/resource/image/tool_op6.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op7.png</normaloff>:/icon/resource/image/tool_op7.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>Function</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op9.png</normaloff>:/icon/resource/image/tool_op9.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>AND</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op10.png</normaloff>:/icon/resource/image/tool_op10.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOperator11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;
padding-left: 16px;
background-color: linear-gradient(rgb(234, 234, 234), rgb(218, 218, 218));
border-color: rgb(204, 204, 204) rgb(204, 204, 204) rgb(181, 181, 181);
</string>
         </property>
         <property name="text">
          <string>M Take K</string>
         </property>
         <property name="icon">
          <iconset resource="../resource.qrc">
           <normaloff>:/icon/resource/image/tool_op11.png</normaloff>:/icon/resource/image/tool_op11.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
